Documentation/getting-started.md

```diff
@@ -0,0 +1,134 @@
+# Getting Started with acbuild
+
+The following guide will walk you through building and running a simple ACI
+with acbuild. The ACI will contain nginx, and be used to host static files.
+
+Note that some of the commands shown here must be run as root.
+
+## Making the ACI
+
+### Starting the build
+
+All acbuild builds have a context, that needs to be explicitly created when the
```

Comment:
All acbuilds happen within a *context*. This build context is explicitly created at `acbuild begin`, and deleted at build completion with `acbuild end`. By default, the context and build are stored in the current working directory.